packages/ui-portal/src/Portal/index.tsx

```diff
@@ -213,20 +212,13 @@ class Portal extends Component<PortalProps, PortalState> {
     return this.DOMNode
   }
 
-  render(): React.ReactPortal | null {
-    if (!canUseDOM) {
-      warn(
-        false,
-        `It looks like you are trying to render <Portal> on the server side, which is not supported!
-         Please be aware that this will result in a no-op!
-         Read more about it here: https://instructure.design/#server-side-rendering`
```

Comment:
If this error is not valid anymore, please remove the warning from the https://instructure.design/#server-side-rendering docs too!